web_duplicate_visibility/static/src/js/duplicate_visibility.js

```diff
@@ -0,0 +1,44 @@
+odoo.define('web.DuplicateVisibility',function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var Sidebar = require('web.Sidebar');
+    var FormView = require('web.FormView');
+
+    var _t = core._t;
+
+    var DuplicateVisibility = FormView.include({
+        /**
+         * Instantiate and render the sidebar if a sidebar is requested
+         * Sets this.sidebar
+         * @param {jQuery} [$node] a jQuery node where the sidebar should be inserted
+         * $node may be undefined, in which case the FormView inserts the sidebar in a
+         * div of its template
+         **/
+        render_sidebar: function($node) {
```

Comment:
Hi, I'm not sure if it's possible but I see that you're overwriting this function (1), instead of overloading and calling _super_. Do you think it's possible ?
something like (if I understood correctly your changes) : 
- call super ;
- remove_items 'Duplicate' if condition is not fulfilled ;

If it's not possible, could you just write a mention saying which line(s) you changed ?

(1) https://github.com/odoo/odoo/blob/9.0/addons/web/static/src/js/views/form_view.js#L204